sdk/core/Azure.Core/src/RequestUriBuilder.cs

```diff
@@ -257,7 +257,13 @@ public void AppendPath(string value, bool escape)
         /// <returns>A string representation of this <see cref="RequestUriBuilder"/>.</returns>
         public override string ToString()
         {
-            var stringBuilder = new StringBuilder();
+            var stringBuilder = new StringBuilder(
+                (Scheme?.Length ?? 0) +
+                3 + // ://
```

Comment:
Is 3 the number of characters or number of bytes? Same with the 10 below.

Reply:
characters.